src/components/calcite-date/calcite-date.tsx

```diff
@@ -210,6 +213,10 @@ export class CalciteDate {
   //  Private Methods
   //
   //--------------------------------------------------------------------------
+  @Watch("value") validateValue(value: string): void {
```

Comment:
Could we update this function to be called `valueWatcher` since it doesn't validate the value?